src/components/ClusterOverview/Capacity/Capacity.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+import * as _ from 'lodash';
```

Comment:
import just the needed function (isNaN) plase

Reply:
done - it's even better to use `Number.isNaN()`